lstchain/version.py

```diff
@@ -1,168 +1,22 @@
-    except (OSError, CalledProcessError):
-        return None
-
-
-def format_git_describe(git_str, pep440=False):
-    """format the result of calling 'git describe' as a python version"""
-
-    if "-" not in git_str:  # currently at a tag
-        formatted_str = git_str
-    else:
-        # formatted as version-N-githash
-        # want to convert to version.postN-githash
-        git_str = git_str.replace("-", ".post", 1)
-        if pep440:  # does not allow git hash afterwards
-            formatted_str = git_str.split("-")[0]
-        else:
-            formatted_str = git_str.replace("-g", "+git")
-
-    # need to remove the "v" to have a proper python version
-    if formatted_str.startswith('v'):
-        formatted_str = formatted_str[1:]
-
-    return formatted_str
-
-
-def read_release_version():
-    """Read version information from VERSION file"""
-    if not path.exists(VERSION_FILE):
-        return 'unknown'
-    with open(VERSION_FILE) as f:
-        return literal_eval(f.read().replace('version=', '', 1))
-
-
-def update_release_version(pep440=False):
-    """Release versions are stored in a file called VERSION.
-    This method updates the version stored in the file.
-    This function should be called when creating new releases.
-    It is called by setup.py when building a package.
-
-
-    pep440: bool
-        When True, this function returns a version string suitable for
-        a release as defined by PEP 440. When False, the githash (if
-        available) will be appended to the version string.
-
-    """
-    version = get_version(pep440=pep440)
-    with open(VERSION_FILE, "w") as outfile:
-        outfile.write(f"version='{version}'")
-        outfile.write("\n")
-
-
-def get_version(pep440=False):
-    """Tracks the version number.
-
-    pep440: bool
-        When True, this function returns a version string suitable for
-        a release as defined by PEP 440. When False, the githash (if
-        available) will be appended to the version string.
-
-    The file VERSION holds the version information. If this is not a git
-    repository, then it is reasonable to assume that the version is not
-    being incremented and the version returned will be the release version as
-    read from the file.
-
-    However, if the script is located within an active git repository,
-    git-describe is used to get the version information.
-
-    The file VERSION will need to be changed manually.
-    """
-
-    raw_git_version = get_git_describe_version()
-    if not raw_git_version:  # not a git repository
-        return read_release_version()
-
-    git_version = format_git_describe(raw_git_version, pep440=pep440)
-
-    return git_version
-
-
-if __name__ == "__main__":
-    print(get_version())
+        from ._dev_version import version
+    except ImportError:
+        from ._version import version
+except Exception:
+    import warnings
+
+    warnings.warn(
```

Comment:
What problems?

The version not being known at all arises if 
a) not git information is available at all (e.g. when doing `pip install https://github.com/<orga>/cta-lstchain/archive/<ref>.tar.gz` instead of `pip install git+https://github.com/<orga>/cta-lstchain@ref`
b) In case of a fork, the tags are not synced from the upstream, so the version is known but off. Fixed by `git fetch upstream; git push origin --tags`

In this regard, this PR does not make a differece. But doing `pip install git+https://github.com/<orga>/cta-lstchain@ref`  will give a correct version.

Reply:
my concern was what happened if somebody installed lstchain from a fork following procedure b). I think the answer is that "tags are not synced from the upstream, so the version is known but off" is the answer and the code will not enter into this exception.

Reply:
my concern was what happened if somebody installed lstchain from a fork following procedure b). I think the answer is that "tags are not synced from the upstream, so the version is known but off" is the answer and the code will not enter into this exception.

Reply:
In any case, the user is warned if this happens and can try the suggested way to install lstchain.